python/samples/SAMPLE_GUIDELINES.md

```diff
@@ -33,7 +33,7 @@ Try to do a best effort to make sure that the samples are incremental in complex
 
 ### **Documentation**
 
-Try to over-document the samples. This includes comments in the code, README.md files, and any other documentation that is necessary to understand the sample.
+Try to over-document the samples. This includes comments in the code, README.md files, and any other documentation that is necessary to understand the sample. We use the guidance from [PEP8](https://peps.python.org/pep-0008/#comments) for comments in the code.
```

Comment:
Below, which I can't comment on because it wasn't changed as part of the PR, we should update where we do the "summary" section from code comments to the style:

```python
"""
This sample shows how to create a chatbot...
"""
```